src/Libraries/RevitNodes/Elements/AdaptiveComponent.cs

```diff
@@ -127,8 +316,7 @@ public List<Point> Locations
         // without breaking replication guides
 
         [IsVisibleInDynamoLibrary(false)]
-        [AllowRankReduction]
-        public static AdaptiveComponent[] ByParametersOnFace(double[][][] uvs, ElementFaceReference faceReference, FamilyType familyType)
+        public static AdaptiveComponent ByParametersOnFace(double[][] uvs, ElementFaceReference faceReference, FamilyType familyType)
```

Comment:
Why is this new method is hidden in the library? Can we take a list of Geometry.UV instead of double?